### PR TITLE
Feature/redesign badges

### PR DESCRIPTION
**Pull request by gazizova-vlada**
Changed badges design and added quick buttons in rows and table views. Changed displayed content in rows and table views.

### CODE REVIEW

File: products/ASC.Files/Client/src/components/Badges.js [hunk 1]
```diff
@@ -19,130 +21,107 @@ const Badges = ({
...
   const showEditBadge = !locked || item.access === 0;
   const isPrivacy = isPrivacyFolder && isDesktopClient;
+  const showActionsEdit = isEditing || isEditingWithFav;
+
+  const iconEdit = showActionsEdit
+    ? "/static/images/file.actions.convert.edit.doc.react.svg"
+    : "/static/images/access.edit.react.svg";
+
+  const contentBadgeVersion =
+    `V.${versionGroup}` > 999 ? "V.999+" : `V.${versionGroup}`;
```
Comment (tarasov-artem) [inline products/ASC.Files/Client/src/components/Badges.js:42]: you must use "Version" language key

Comment (gazizova-vlada) [reply products/ASC.Files/Client/src/components/Badges.js]: Added a new VersionBadge key and used it.

Comment (gazizova-vlada) [reply products/ASC.Files/Client/src/components/Badges.js]: > you must use "Version" language key

Added a new VersionBadge key and used it.

Comment (gazizova-vlada) [reply products/ASC.Files/Client/src/components/Badges.js]: Added a new VersionBadge key and used it.

**Review by tarasov-artem — APPROVED** [review]

File: products/ASC.Files/Client/src/pages/Home/Section/Body/RowsView/FilesRowContent.js [hunk 2]
```diff
@@ -64,22 +87,19 @@ const FilesRowContent = ({
...
     fileExst,
     filesCount,
     foldersCount,
     providerKey,
-    access,
     title,
   } = item;
 
-  const withAccess = isAdmin || access === 0;
+  updatedDate = moment(new Date(updatedDate)).format("MM/D/YYYY	hh:mm A");
```
Comment (ilyaoleshko) [inline products/ASC.Files/Client/src/pages/Home/Section/Body/RowsView/FilesRowContent.js:102]: MomentJS is to heavy for this. If you need remove comma from date string just replase it inside  "getStatusByDate" and "getTableStatusByDate" finctions at withContent HOC`s.

File: products/ASC.Files/Client/src/pages/Home/Section/Body/TableView/sub-components/DateCell.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
 import React from "react";
 import { StyledText } from "./CellStyles";
+import moment from "moment";
 
 const DateCell = ({ create, updatedDate, createdDate, sideColor }) => {
-  const date = create ? createdDate : updatedDate;
+  const date = moment(new Date(create ? createdDate : updatedDate)).format(
```
Comment (ilyaoleshko) [inline products/ASC.Files/Client/src/pages/Home/Section/Body/TableView/sub-components/DateCell.js:6]: As in comment above. Data from "getTableStatusByDate"

File: products/ASC.Files/Client/src/pages/Home/Section/Body/RowsView/FilesRowContent.js (hunk 2 above)
Comment (gazizova-vlada) [reply products/ASC.Files/Client/src/pages/Home/Section/Body/RowsView/FilesRowContent.js]: Removed momentJS from FilesRowContent component. Combined the getStatusByDate and getTableStatusByDate methods into the getStatusByDate method in the withContent HOC`s. Added parameters for the toLocaleString method in the getStatusByDate method. Replaced the comma from date string inside the getStatusByDate method.

File: products/ASC.Files/Client/src/pages/Home/Section/Body/TableView/sub-components/DateCell.js (hunk 3 above)
Comment (gazizova-vlada) [reply products/ASC.Files/Client/src/pages/Home/Section/Body/TableView/sub-components/DateCell.js]: Removed momentJS from DateCell component.

**Review by ilyaoleshko — APPROVED** [review]

**Review by tarasov-artem — APPROVED** [review]

**Pull request merged**